Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -482,3 +483,39 @@ def test_is_mac_address():
 
     assert(is_mac_address(mac_address_false) is False)
     assert(is_mac_address(mac_address_true))
+
+
+def test_return_error(mocker):
+    from CommonServerPython import return_error
+    err_msg = "Testing unicode Ё"
+    outputs = {'output': 'error'}
+    expected_error = {
+        'Type': entryTypes['error'],
+        'ContentsFormat': formats['text'],
+        'Contents': err_msg,
+        "EntryContext": outputs
+    }
+
+    # Test command that is not fetch-incidents
+    mocker.patch.object(demisto, 'command', return_value="test-command")
+    mocker.patch.object(sys, 'exit')
+    mocker.spy(demisto, 'results')
+    return_error(err_msg, '', outputs)
+    assert str(demisto.results.call_args) == "call({})".format(expected_error)
+
+    # Test fetch-incidents
```

Comment:
Better to separate this one to another test, each test module in python should test on flow of functionality - thats the best practice